Makefile

```diff
@@ -119,16 +119,17 @@ core-run: generate fmt vet manifests
 core-install: manifests
 	kubectl apply -f charts/vela-core/crds/
 	kubectl apply -f charts/vela-core/templates/definitions/
+	kubectl apply -f charts/third_party/prometheus
 
 # Uninstall CRDs and Definitions of Vela Core from a cluster, this is for develop convenient.
 core-uninstall: manifests
 	kubectl delete -f charts/vela-core/crds/
 	kubectl delete -f charts/vela-core/templates/definitions/
+	kubectl delete -f charts/third_party/prometheus
 
 # Generate manifests e.g. CRD, RBAC etc.
 manifests: controller-gen
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=charts/vela-core/crds
-	rm charts/vela-core/crds/_.yaml
```

Comment:
don't delete it

Reply:
already reverted

Reply:
**`charts/vela-core/crds/_.yaml`** is automatically generated， I don't know how to prevent it from generating.